### ISSUE TITLE
Few bugs with the profile

### ISSUE THREAD

**Issue by Blandine-AA** [hhaccessibility/hhaccessibility.github.io#538]
When starting as a new user, I found the following issues:

Distance default value is set to 0, but minimal allowed value is 0,01. This should be the default value
- [ ] Set Distance default to 0.01 in the Profile

If clicking on Select All to chose the criteria, without having clicked on any checkbox, the save button stays disabled.
- [x] Activate the save button when changes are made, regardless of how
- [ ] Profile values such as Distance and Location Search should not be automatically updated when user changes it on the other pages (Home, Location search...)

**Comment by passionInfinite** [comment]
@joshi1983  is this issue fixed? If yes close it.

**Comment by joshi1983** [comment]
@Rubynguyen2910 how are you doing with this?  You've been quiet for a while.

**Comment by joshi1983** [comment]
One of the subtasks from this is covered by #607.  That includes the following text from this issue:

```
If clicking on Select All to chose the criteria, without having clicked on any checkbox, the save button stays disabled.

  Activate the save button when changes are made, regardless of how```

**Comment by joshi1983** [comment]
The best change I can see for "Profile values such as Distance and Location Search should not be automatically updated when user changes it on the other pages (Home, Location search...)" is to remove those from the profile.  The reason is that those features are useless if they're not updated by the location search and home pages.  The home and location search is where the best user interface is for showing and making use of the search radius and search location.

The profile is already has so much content that people are complaining about not finding features like Location Add on it and complaining about there being too many questions on it.  Removing the location search radius and keywords would also shorten the page and bring more important content to their attention.

**Comment by M4ttoF** [comment]
After discussing this in the meeting, we should change default value to 15km rather than 0.01km

**Comment by stephanosm** [comment]
Also the increment to be 1Km not 0.01Km

**Comment by joshi1983** [comment]
This will be resolved by #695 which involves completely removing 'Search Location', and this distance from the profile.  For this reason, I'm closing the issue.
